js_modules/dagster-ui/packages/ui-components/src/components/Toaster.tsx

```diff
@@ -5,9 +5,9 @@ import {BaseButton} from './BaseButton';
 import {Box} from './Box';
 import {Colors} from './Color';
 import {Icon, IconName} from './Icon';
-import {FontFamily} from './styles';
+import styles from './css/Toaster.module.css';
 
-export {Toaster} from 'sonner';
+export {Toaster, toast} from 'sonner';
```

Comment:
Also exporting this to allow callers to use the `toast` API, e.g. to dismiss toasts by ID.